infra/feast-operator/test/e2e/e2e_test.go

```diff
@@ -127,6 +127,117 @@ var _ = Describe("controller", Ordered, func() {
 			}
 			EventuallyWithOffset(1, verifyControllerUp, time.Minute, time.Second).Should(Succeed())
 
+			By("deploying the Simple Feast Custom Resource to Kubernetes")
+			cmd = exec.Command("kubectl", "apply", "-f",
+				"test/testdata/feast_integration_test_crs/v1alpha1_default_featurestore.yaml")
+			_, cmdOutputerr := utils.Run(cmd)
+			ExpectWithOffset(1, cmdOutputerr).NotTo(HaveOccurred())
+
+			By("validate the feast registry pod is up and running state.")
+			verifyFeastControllerUp := func() error {
+				// Get pod name
+
+				cmd := exec.Command("kubectl", "get",
+					"pods", "-l", "feast.dev/service-type=registry",
+					"-o", "go-template={{ range .items }}"+
+						"{{ if not .metadata.deletionTimestamp }}"+
+						"{{ .metadata.name }}"+
+						"{{ \"\\n\" }}{{ end }}{{ end }}",
+					"-n", "default",
+				)
+
+				podOutput, err := utils.Run(cmd)
+				ExpectWithOffset(2, err).NotTo(HaveOccurred())
+				podNames := utils.GetNonEmptyLines(string(podOutput))
+				if len(podNames) != 1 {
+					return fmt.Errorf("expect 1 feast registry pod is running, but got %d", len(podNames))
+				}
+				controllerPodName = podNames[0]
+				ExpectWithOffset(2, controllerPodName).Should(ContainSubstring("default-feast-cr"))
+
+				// Validate pod status
+				cmd = exec.Command("kubectl", "get",
+					"pods", controllerPodName, "-o", "jsonpath={.status.phase}",
+					"-n", "default",
+				)
+				status, err := utils.Run(cmd)
+				ExpectWithOffset(2, err).NotTo(HaveOccurred())
+				if string(status) != "Running" {
+					return fmt.Errorf("registry pod in %s status", status)
+				}
+				return nil
+			}
+			EventuallyWithOffset(1, verifyFeastControllerUp, time.Minute, time.Second).Should(Succeed())
+
+			By("Check if the feast client - kubernetes config map exists.")
+			verifyFeastClientConfigMapExists := func() error {
+				// Get pod name
+
+				cmd := exec.Command("kubectl", "get",
+					"configmap", "feast-default-feast-cr-client",
+					"-n", "default", "--no-headers",
+				)
+
+				cmOutput, err := utils.Run(cmd)
+				ExpectWithOffset(2, err).NotTo(HaveOccurred())
+				configMap := utils.GetNonEmptyLines(string(cmOutput))
```

Comment:
it would be good to validate the contents of the client configmap are as expected